devtools/conda-envs/openff.yaml

```diff
@@ -31,6 +31,7 @@ dependencies:
   - rdkit
   - geometric>=0.9.3
   - torsiondrive
+  - intel-openmp!=2019.5-281
```

Comment:
Do you want to skip all of 2019.5 or just 281?

Reply:
It turns out that "2019.5-281" doesn't register with conda as a version, so now I'm skipping "2019.5".